Makefile

```diff
@@ -1,4 +1,4 @@
-COVERAGE_THRESHOLD ?= 59.8
+COVERAGE_THRESHOLD ?= 60.360.360.360.360.360.3
```

Comment:
vim loves to do that with me as punishment for only editing Makefiles with it and using a real editor otherwise ;)